linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/bash/shared.sh

```diff
@@ -1,6 +1,9 @@
 # platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Virtualization 4,multi_platform_wrlinux,multi_platform_ol,multi_platform_sle
 
```

Comment:
We can remove the jinja if else by introducing a jinja variable for the `sle` difference. Note the only difference is that the aide path is `bin` and `sbin` in the other.

So something like
```
{{%if product in ["sle12", "sle15"] %}}
    {{% set aide_bin_path="bin" %}}
{{% else %}}
    {{% set aide_bin_path="sbin" %}}
{{% endif %}}
```

Reply:
Just a suggestion from the peanut gallery, but I would make this the full path. On Ubuntu we use`/usr/bin/aide.wrapper`. :-)

Reply:
it can be, then it probably needs some extra work around the grep command, which uses escaped version of the path

Reply:
Just curious, but why would grep need escaping? 

```
$ grep  '/var/log/audit'  linux_os/guide/system/auditing/auditd_configure_rules/logs_group_ownership_var_log_audit/bash/shared.sh 
    chown :adm /var/log/audit/
    sed -i -e '$alog_file = /var/log/audit/audit.log' /etc/audit/auditd.conf
```

grep has no problems with the `/` character and the other content there will prevent it from being interpreted as a path.